author.rb

```diff
@@ -0,0 +1,18 @@
+# Create an Author class with an association to the Item class.
+require './classes/item'
+class Author < Item
```

Comment:
Author class is not inheriting from Item class according to the diagram given.

Reply:
Your review is so perfect
Great 👍 
Thanks a lot for your help!